Av1an/compose.py

```diff
@@ -86,9 +86,9 @@ def compose_ffmpeg_pipe(a: Args) -> str:
     Gets the ffmpeg command with filters that pipes into the encoder
 
     :param a: the Args
-    :return: the ffmpeg command as a string
+    :return: the ffmpeg command as a list
     """
-    return f'ffmpeg -y -hide_banner -loglevel error -i - {a.ffmpeg} {a.pix_format} -bufsize 50000K -f yuv4mpegpipe -'
+    return ['ffmpeg', '-y', '-hide_banner', '-loglevel', 'error', '-i', '-', *a.ffmpeg_pipe]
```

Comment:
Since every other command is a tuple, this one should be too. You should also update the return type in the function signature.
```suggestion
    return ('ffmpeg', '-y', '-hide_banner', '-loglevel', 'error', '-i', '-', *a.ffmpeg_pipe)
```